frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTitle.tsx

```diff
@@ -16,7 +16,7 @@ const PipelineRunTitle: React.FC<PipelineRunTitleProps> = ({ run }) => {
 
   return (
     <Split hasGutter>
-      <SplitItem>{run.name}</SplitItem>
+      <SplitItem className="pf-v5-u-text-break-word">{run.name}</SplitItem>
```

Comment:
`PipelineDetails` uses `Truncate`. The two pages behave differently for long names.

Reply:
I will use Truncate here.